projects/ngx-drag-to-select/src/lib/select-container.spec.ts

```diff
@@ -86,6 +89,22 @@ describe('SelectContainerComponent', () => {
     expect(fixture.componentInstance.selectItems.first.calculateBoundingClientRect).toHaveBeenCalled();
   });
 
+  it('should update its selection when selectable items change', (done) => {
+    selectContainerInstance.selectItems((item: SelectItemValue) => item.id === 1 || item.id === 2);
+    selectContainerInstance.itemDeselected.subscribe((item: SelectItemValue) => {
+      expect(item).toEqual({ id: 1 });
+    });
+
+    selectContainerInstance.select.subscribe((items) => {
+      expect(items).toEqual([{ id: 2 }]);
+      done();
+    });
+
+    const previousData = testComponent.data$.value;
+    testComponent.data$.next([previousData[1], previousData[2], { id: 4 }]);
+    fixture.detectChanges();
```

Comment:
Add some newlines here
```suggestion
    const previousData = testComponent.data$.value;
    
    testComponent.data$.next([previousData[1], previousData[2], { id: 4 }]);
    
    fixture.detectChanges();
```